streams/src/test/java/org/apache/kafka/streams/integration/StandbyTaskEOSIntegrationTest.java

```diff
@@ -81,17 +77,7 @@ public class StandbyTaskEOSIntegrationTest {
     private final static int KEY_0 = 0;
     private final static int KEY_1 = 1;
 
-    @SuppressWarnings("deprecation")
-    @Parameterized.Parameters(name = "{0}")
-    public static Collection<String[]> data() {
-        return asList(new String[][] {
-            {StreamsConfig.EXACTLY_ONCE},
-            {StreamsConfig.EXACTLY_ONCE_V2}
-        });
-    }
-
-    @Parameterized.Parameter
-    public String eosConfig;
+    public String eosConfig = StreamsConfig.EXACTLY_ONCE_V2;
```

Comment:
I think it would make sense to replace `eosConfig` with `StreamsConfig.EXACTLY_ONCE_V2` in these tests. I already imagine future me wondering about this useless indirection.

Reply:
Good point, Done